### ISSUE TITLE
Notification Counters for Channels

### ISSUE THREAD

**Issue by ShendMikullovci** [serge-web/serge#1824]
Hello everyone!

I was wondering when and how do notification counters disappear from the channel tabs in-game? Clicking on the tabs doesn't do the trick:

[5ded5262-3085-4a05-b4de-61b4a00bc023.webm](https://user-images.githubusercontent.com/85491794/188909524-e4b8f8ee-23b4-4cc1-ac90-27be6db8d3ce.webm)

Once the user sees the message, the notification is no longer useful. I suggest we either add mark read/unread buttons, or have the counters disappear after a few seconds of the tab being consulted.

**Comment by IanMayo** [comment]
We may be missing some mark as read/mark as unread functionality.  For channels with templates, new messages are shown collapsed by default.  So, the user physically expands them, and then we treat that message as read.  But, for chat channels, we don't have the concept of "read".  _Maybe_ we need it.   In the past, players do seem to have made use of marking a message as "unread" as a way of keeping a to-do list of things that need acting upon.

**Comment by ShendMikullovci** [comment]
Here's a potential look:

[568c8042-9e67-4abb-b3e2-9f82491e69e9.webm](https://user-images.githubusercontent.com/85491794/188920606-b6265ef5-f79f-4919-a4a2-d010a74e3182.webm)

**Comment by IanMayo** [comment]
Yes, that works.  I _guess_ the icon could go inside the message block, since its part of the message.  I also _guess_ we could do with a "mark all as read" button at the top.

But, what do we do about the initial "read" state.  Are new chat messages marked as "Read" or "Unread".  My first thought is that they would be "REad", since the player can see them.  But, if we treat "Read" as "Acknowledged", then they would be "Unread" - and the player would click them individually to acknowledge them, or use "Mark all as read" to indicate they're "up-to-speed" on that channel.
